lib/connect.c

```diff
@@ -1092,9 +1092,11 @@ static void nosigpipe(struct Curl_easy *data,
   int onoff = 1;
   if(setsockopt(sockfd, SOL_SOCKET, SO_NOSIGPIPE, (void *)&onoff,
                 sizeof(onoff)) < 0) {
+#if !defined(CURL_DISABLE_VERBOSE_STRINGS)
     char buffer[STRERROR_LEN];
     infof(data, "Could not set SO_NOSIGPIPE: %s",
           Curl_strerror(SOCKERRNO, buffer, sizeof(buffer)));
+#endif
```

Comment:
This seems unrelated?

Reply:
Yes, this is unrelated. It fixes the warning unused local variable `buffer` when I compile with cmake option `-DCURL_DISABLE_VERBOSE_STRINGS=ON`. I can remove this change it from this PR.